settings.gradle.kts

```diff
@@ -35,6 +35,9 @@ plugins {
 
 includeBuild("hedera-platform")
 
+// If you turn this is on, 'platform-sdk' modules are built from source
+// includeBuild("platform-sdk")
```

Comment:
See: https://github.com/hashgraph/hedera-services/pull/7330#discussion_r1246756195

This is the switch to turn on the "true" integration between "hedera-node" and "platform-sdk". Once this is on,  "hedera-node" will build against the sources of "platform-sdk" (instead of the published version). 

I commented it out in this PR, because the code (right now) won't compile. There are changes in "platform-sdk" right now that are not compatible.